qcommon/src/main/java/jake2/qcommon/MSG.java

```diff
@@ -31,15 +31,10 @@
     // writing functions
     //
 
-    //ok.
-    public static void WriteChar(sizebuf_t sb, int c) {
-        sb.data[SZ.GetSpace(sb, 1)] = (byte) (c & 0xFF);
-    }
-
     //ok.
     public static void WriteChar(sizebuf_t sb, float c) {
 
-        WriteChar(sb, (int) c);
+        WriteByte(sb, (int) c);
```

Comment:
identical

Reply:
what does this mean?

Reply:
these functions `WriteChar`/`WriteByte` did exactly the same thing, so i removed one